Documentation/Signoffs/Create DAQ Subsystem.md

```diff
@@ -0,0 +1,109 @@
+
+
+
+### Resolution and Sample Rate:
+
+To prove that the NI USB-6215 system can successfully sample accurately enough for all of the sensors, the team finds values to determine if this specific system is fast enough to determine real-time values. Finding the maximum values of sample rate and I/O pin currents, as well as the resolution, can show how accurate and precise the sensors are measuring.
+
+The sample rate is given in the data sheet of the system as 250kS/s, which exceeds constraint 18 by a significant amount.  This is very high compared to most systems and allows the system to capture measurements at a greater accuracy, along with a better resolution.
+
+
+Use the following equation to determine the precision that the USB-6215 provides at 16 bits of resolution: 
+
+![image](https://github.com/Dylan2432/Capstone1_Team3_EV-Motorcycle-Chassis-Dynamometer/assets/157453598/720d4f84-d0a3-44b4-ab01-e97a1ae3008c)
+
+
+  Where VD is the full-scale input range of the logger, which is 10 V, VS is the full-scale range of the measured signal, which is 10 V, E is the engineering units that represent VS, Resolution represents the smallest change the DAQ can measure, and B is 0 as the logger is unipolar meaning it will measure from 0-10 V.
+
+
+We can verify the accuracy of the equation through an example. Say we have a DAQ with an input range of 0-10 volts, a sensor that measures current from 0-100 amps with an output range of 0-10 volts, the sensor was unipolar and the DAQ has a bit resolution of 10 bits. 10 bits represent 1024 locations that can be addressed. In this example our resolution is 0.09765625 A/location which is the smallest change the DAQ can measure. Then if we multiply our resolution by the number of locations the DAQ can provide, 1024, we get back to 100 A.
+
+
+
+  Our E value was chosen as higher than the expected value to prepare for a worst-case scenario. A lower E value would lead to more precision.
+
+  For voltage, our E is 100 VDC, which allows our precision to be 0.001526 V. This means the logger will respond to voltage changes of 0.001526V or more. 1% of E is 1; therefore precision of 0.001526 meets and exceeds the limitations of constraint 19.
+
+
+For current, our E is 50 A, which allows the precision to be 0.000763 A. 1% of E is 0.5; therefore precision of 0.000763 meets and exceeds the limitations of constraint 19.
+
+
+  For RPM, our E is 1000 RPM, this is based on the max rpm of the rear wheel being 700.69. This means the precision is 0.01526. 1% of E is 10; therefore precision of 0.01526 meets and exceeds the limitations of constraint 19.
+
+  For torque our E is 10 Nm, this is based on the max torque of the rear wheel being 4.995 Nm. This means the precision is 0.0001526 Nm. 1% of E is 0.1; therefore precision of 0.0001526 meets and exceeds the limitations of constraint 19.
```

Comment:
Again, we are not reading the speed of the back wheel but the speed of the roller. What is the gear ratio between the back wheel and roller? How does this affect your calculation?